crates/services/p2p/src/p2p_service.rs

```diff
@@ -180,7 +180,7 @@ impl FuelP2PService {
 
         // configure and build P2P Service
         let (transport_function, connection_state) = build_transport_function(&config);
-        let tcp_config = tcp::Config::new().port_reuse(true);
+        let tcp_config = tcp::Config::new();
```

Comment:
What's the reason for removing `.port_reuse(true)`?

Reply:
This option does nothing now, since the port reuse policy is now decided on a per-connection basis by the behaviour.  It has [deprecated](https://github.com/libp2p/rust-libp2p/blob/master/transports/tcp/src/lib.rs#L185-L189) since `0.42.0`